assets/src/sass/helpers/mixins/_mixins-links.scss

```diff
@@ -157,7 +157,7 @@
 	&.category-public-policy::before {
 		background-color: var(--wp--preset--color--bright-green);
 	}
-	&.category-our-community::before {
+	&.category-community::before {
```

Comment:
Would it help the migration period if we supported both classes? Or is only the new class valid?

Reply:
No, I actually created the "our-community" one before I realized the other one existed. And for the migration period, worst case is that the dots are all blue instead of multi-colored.